### PR TITLE
chore(rxMultiSelect): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-356

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxMultiSelect/scripts/rxMultiSelect.js [hunk 1]
```diff
@@ -164,59 +146,4 @@ angular.module('encore.ui.rxMultiSelect', ['encore.ui.rxSelectFilter'])
...
- * titleized version of `value`.
- *
- * <pre>
- * <rx-select-option value="DISABLED">Disabled</rx-select-option>
- * </pre>
- *
- * @param {String} value The value of the option. If no transcluded content is provided,
- *                       the value will also be used as the option's text.
- */
-.directive('rxSelectOption', function (rxDOMHelper) {
```
Comment (CITguy) [inline src/rxMultiSelect/scripts/rxMultiSelect.js]: moved to `src/rxMultiSelect/scripts/rxSelectOption.js`

File: src/rxMultiSelect/scripts/rxMultiSelect.page.js [hunk 2]
```diff
@@ -214,51 +210,4 @@ exports.rxMultiSelect = {
...
-       @returns {rxMultiSelect} Page object representing the _first_ rxMultiSelect object found on the page.
-    */
-//     main: (function () {
-//         rxMultiSelect.rootElement = {
-//             get: function () { return $('#rxMultiSelect'); }
-//         };
-//         return Page.create(rxMultiSelect);
-//     })()
-
-// };
```
Comment (CITguy) [inline src/rxMultiSelect/scripts/rxMultiSelect.page.js]: removed commented boilerplate code

**Comment by bry4n** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**